### DISCUSSION TITLE
Upgrading to strimzi 0.26.0

### DISCUSSION THREAD

**Discussion by opsocomusr** [strimzi/strimzi-kafka-operator#5856] [Q&A] [answered]
When we upgrade from Strimzi 0.25 to 0.26, we are trying to use the same version of kafka (2.8.0) that we used in strimzi 0.25. The kafka pods gets rolled as expected with kafka:0.26.0-kafka-2.8.0 image but it gets stuck while rolling the entity operator pod.

The user-operator and topic-operator logs shows it is trying to use kafka version 3.0.0 instead of 2.8.0 which is the defined version in the strimzi custom resource. How to configure entity operator to use the kafka version 2.8.0 in strimzi 0.26?

**Comment by scholzj** [comment, marked as answer]
The Kafka version you choose applies to the Kafka brokers (or Connect nodes etc.). But it does not apply to the operators and other components. They always use only one Kafka version - usually the latest.
